pkg/services/accesscontrol/dualwrite/reconciler.go

```diff
@@ -93,11 +94,11 @@ func NewZanzanaReconciler(cfg *setting.Cfg, client zanzana.Client, store db.DB,
 				client,
 			),
 		},
-		globalReconcilers: []globalReconciler{
-			newGlobalReconciler(
+		globalReconcilers: []resourceReconciler{
+			newResourceReconciler(
 				"fixed role pemissions",
 				fixedRolePermissionsCollector(store),
-				zanzanaCollector([]string{zanzana.RelationAssignee}),
+				zanzanaCollector(zanzana.RelationsFolder),
```

Comment:
This constant have all possible relations we can have for resources, a bit misleading

Reply:
Why do we change relation from "assignee" to "get/update/greate/delete"?